jscomp/ext/ext_filename.ml

```diff
@@ -302,7 +302,27 @@ let rel_normalized_absolute_path from to_ =
           Ext_string.parent_dir_lit xs in
     go paths1 paths2
 
-(*TODO: could be hgighly optimized later 
+(**
+  Instead of splitting a string, returns a list of indices where directory names start
+ *)
+let split_aux_idx x =
+  let l = String.length x in
+  let ds = Filename.dir_sep.[0] in (*dir_sep as char*)
```

Comment:
can you look up `Filename.dir_sep.[0]` above, it should already be singled out as a toplevel binding